src/main/java/codeu/controller/ProfileListServlet.java

```diff
@@ -0,0 +1,61 @@
+package codeu.controller;
+
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.store.basic.MessageStore;
+import codeu.model.store.basic.UserStore;
+import codeu.model.data.User;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/**
+ * Created by gavinlifrieri on 6/12/18.
+ */
+public class ProfileListServlet extends HttpServlet {
+
+
+    /**
+     * Store class that gives access to Conversations.
+     */
+    private ConversationStore conversationStore;
```

Comment:
Is `conversationStore` or `messageStore` used? If not you can probably get rid of them.